pyanaconda/ui/tui/spokes/storage.py

```diff
@@ -324,10 +324,10 @@ def run_dasdfmt(self, to_format=None):
 
             if self.data.zerombr.zerombr:
                 # unformatted DASDs
-                unformatted += make_unformatted_dasd_list([d for d in self.data.ignoredisk.onlyuse])
+                unformatted += make_unformatted_dasd_list([d.name for d in getDisks(self.storage.devicetree)])
```

Comment:
``getDisks()`` gives a different list of devices so I think this change should be mentioned in the commit message.